package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nasa-jpl/seq-json-schema",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
I think this merits at least a minor version increment instead of just a patch, given we're changing the `type` values.

Reply:
Sure, I made it 1.2.0